ui/src/ScholarReader.tsx

```diff
@@ -313,6 +321,28 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
         if (symbols.length >= 1) {
           const mathMl = await api.mathMlForArxivId(this.props.paperId.id);
           this.setMathMl(mathMl);
+
+          /**
+           * Build a mapping from symbol to all possible parent matches
+           */
+          const symbolMatchSet: SymbolMatchSet = {};
+          symbols.forEach(s => {
+            symbolMatchSet[s.id] = new Set(
+              selectors.matchingSymbols(s, symbols, mathMl)
+              .map(m => {
```

Comment:
```suggestion
              .map(s => {
```

Or `symbol` or something that conveys the meaning of the list item